web/src/main.tsx

```diff
@@ -42,12 +45,17 @@ const GlobalNav = () => {
 
 const App = () => {
   return (
-    <div class="max-w-5xl ma">
+    <div class='max-w-5xl ma'>
       <h1>stackablectl, web edition</h1>
       <GlobalNav />
       <Routes>
-        <Route path="/listeners" component={Listeners} />
-        <Route path="/" component={Home} />
+        <Route
+          path='/product-clusters/:namespace/:name/connect'
```

Comment:
Please feel free to ignore this comment, I have no clue of this kind of stuff. I was just thinking what happens when you have a ZK `simple` and  HDFS `simple`? Will the paths clash?
WDYT of `/product-clusters/:namespace/:product/:name/connect`?

Reply:
They already do in practice, since they'd up generating the same discovery config.